src/Testing/Http/TestResponseSenderInitializer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Testing\Http;
+
+use Tempest\Container\CanInitialize;
+use Tempest\Container\Container;
+use Tempest\Http\Response;
+use Tempest\Http\ResponseSender;
+
+final class TestResponseSenderInitializer implements CanInitialize
+{
+    public function canInitialize(string $className): bool
+    {
+        return in_array($className, [Response::class, TestResponse::class]);
+    }
+
```

Comment:
On mobile, but shouldn't you also implement Initializer? Not sure

Reply:
Somewhere along the way `CanInitialize` got modified to extend `Initializer` so that now throws errors. I meant to track down when that happened.

Reply:
I refactored it, you now have to implement both. Please rebase on main :)